openedx/core/djangoapps/cors_csrf/admin.py

```diff
@@ -1,7 +1,11 @@
-"""Manage cross-domain configuration. """
+"""
+Manage cross-domain configuration.
+"""
+
 from django.contrib import admin
 from config_models.admin import ConfigurationModelAdmin
-from cors_csrf.models import XDomainProxyConfiguration
+
+from .models import XDomainProxyConfiguration
```

Comment:
That's my preference as well.  Explicit relative imports within a djangoapp or lib, absolute outside.